content/docs/setup/kubernetes/download/index.md

```diff
@@ -51,3 +51,51 @@ To use the Istio release Helm chart repository, add the Istio release repository
 {{< text bash >}}
 $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
 {{< /text >}}
+
+## `istioctl` auto-completion
+
+If you are using Bash or Zsh, `istioctl` includes a tab completion file that you can use. 
+
+### istioctl on Linux
+
```

Comment:
would it be interesting to have on Linux vs on OSX as 2 tabs?  see helm install page for an example.

Reply:
I would totally remove the Linux portion of the title and change it to something like `Installing the completion file`. This section then is the same for all environments. You can say that for MacOS, that `bash-completion` is needed and refer to the following section.

Reply:
@ tabs for now, Linux, Mac, and then a 3rd when zSH comes online. See https://kubernetes.io/docs/tasks/tools/install-kubectl/#optional-kubectl-configurations

Reply:
I would combine the below prerequisite and then the install (swapping order) into 3 tabs.

1) Linux
2) MacOS bash
3) Zsh